cli/command/network/create.go

```diff
@@ -87,34 +95,38 @@ func newCreateCommand(dockerCli command.Cli) *cobra.Command {
 func runCreate(ctx context.Context, dockerCli command.Cli, options createOptions) error {
 	client := dockerCli.Client()
 
-	ipamCfg, err := consolidateIpam(options.ipamSubnet, options.ipamIPRange, options.ipamGateway, options.ipamAux.GetAll())
-	if err != nil {
-		return err
-	}
-
-	var configFrom *network.ConfigReference
-	if options.configFrom != "" {
-		configFrom = &network.ConfigReference{
-			Network: options.configFrom,
-		}
-	}
-	resp, err := client.NetworkCreate(ctx, options.name, types.NetworkCreate{
+	createOpts := types.NetworkCreate{
 		Driver:  options.driver,
 		Options: options.driverOpts.GetAll(),
 		IPAM: &network.IPAM{
 			Driver:  options.ipamDriver,
-			Config:  ipamCfg,
 			Options: options.ipamOpt.GetAll(),
 		},
 		Internal:   options.internal,
-		EnableIPv6: options.ipv6,
```

Comment:
Given that this is set to `nil` if the flag didn't change; is there a need to do the check below?